lib/tooling/base-tool.ts

```diff
@@ -30,18 +30,29 @@ import _ from 'underscore';
 import * as exec from '../exec';
 import {logger} from '../logger';
 import * as utils from '../utils';
+import {ToolEnv, ToolInfo} from './base-tool.interface';
```

Comment:
This was the min needed to get it working. I'll properly do this before merging the PR